.github/workflows/detect_changes.yml

```diff
@@ -0,0 +1,34 @@
+name: Detect Dependencies Changes
+
+on:
+  pull_request:
+    types: [opened, synchronize]
+
+jobs:
+  update-deps:
+    runs-on: ubuntu-latest
```

Comment:
Oh @ParthivNaresh to disambiguate this from the "Update Dependencies" bot which runs every hour, could you rename this?

I would call it "check-dependencies-updated"